content/blog/embrace-kubernetes-part1/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "It’s Time to Embrace Kubernetes! Really? (Part 1)"
+date: 2020-12-11
+meta_desc: "How to evaluate your choices when choosing Kubernetes as your application platform"
+meta_image: embrace-k8s.png
+authors:
+    - lee-briggs
+tags:
+    - kubernetes
+
+---
+
+When you’re considering whether or not to implement Kubernetes, perhaps the first question to ask yourself is do you need it at all?
+
+The point of any technology isn’t the technology itself. The point is whether you can use it to reduce the barrier of entry for application developers so they can get features from their machines to your customers as quickly and easily as possible. Do you already have a solution that works well? If you do, why do you want to change it? Making such a radical change in your technology is potentially quite dangerous so what’s your motivation?
+
+It might be wiser to stay with the solution you have and improve it. Also, don't fall into the trap of believing that any technology, not just Kubernetes, will be a magical answer to all your problems, whether organizational or technical. There’s no such thing.
+
+In this blog post we’ll talk about how to evaluate your own situation to see if Kubernetes is a good fit. We’ll see that the question isn’t that simple to answer and there are a lot of variables to consider. In the next blog post, we’ll talk about a situation where Kubernetes can be a good fit and how to start your first Kubernetes project.
+
+Note that these blog posts assume you already have some familiarity with Kubernetes. If you need some introductory material, the [Kubernetes web site](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/) is a good place to start.
+
+<!--more-->
+
+## Why is Kubernetes So Popular?
+
+Kubernetes, an open-source system for deploying, scaling, and managing containerized applications, is currently the last step in an evolutionary chain that began with private data centers and, from there, went to cloud VMs, then to containers and then to container orchestrators. Kubernetes was developed by Google. Version 1.0 was released in 2015 and the latest version was released in October 2020.
```

Comment:
```suggestion
Kubernetes, an open-source system for deploying, scaling, and managing containerized applications, is currently the last step in an evolutionary chain that began with private data centers and, from there, went to cloud VMs, then to containers and then to container orchestrators. If you are looking to ship applications to the cloud continuously, containers are a great way to do that, and Kubernetes is the technology responsible for running them. Kubernetes was initially developed by Google and has since had an enormous community grow up around it well beyond its Google roots. Version 1.0 was released in 2015 and the latest version was released in October 2020.
```